schema/dereferenced-release-schema.json

```diff
@@ -22644,6 +22707,22 @@
             "description": "The single amendment object has been deprecated in favour of including amendments in an amendments (plural) array.",
             "deprecatedVersion": "1.1"
           }
+        },
+        "finalStatus": {
+          "title": "Award final status",
+          "description": "The final status of the award, from the closed [awardFinalStatus](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#award-final-status) codelist.",
```

Comment:
```suggestion
          "description": "The final status of the award, using the closed [awardFinalStatus](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#award-final-status) codelist.",
```

Reply:
Done in https://github.com/open-contracting/standard/pull/1648/commits/4bab921b7905faf32c1a52101b733deb20ea7eb5